azure/templates/pds-tests.yml

```diff
@@ -45,6 +45,9 @@ steps:
     displayName: Expose common variables
 
   - bash: |
+      export APIGEE_API_TOKEN="$(secret.AccessToken)"
+      export OAUTH_BASE_URI="https://$(ENVIRONMENT).api.service.nhs.uk"
+      export OAUTH_PROXY="oauth2"
```

Comment:
looks like these were duplicated here and line 62

Reply:
in pds-tests.yml  they re-export those variables before each of the new test-bases, so it'll be to ensure each test has the correct variables in each respective container